tests/end_to_end/test_sql_injection/test_sql_injection.py

```diff
@@ -0,0 +1,74 @@
+import os
+import json
+import re
+import pytest
+from datetime import datetime
+from tempfile import gettempdir
+from time import sleep
+
+from wazuh_testing import end_to_end as e2e
+from wazuh_testing import event_monitor as evm
+from wazuh_testing.tools import configuration as config
+from wazuh_testing.tools.time import parse_date_time_format
```

Comment:
```suggestion
from wazuh_testing.tools.time import parse_date_time_format
from wazuh_testing import end_to_end as e2e
from wazuh_testing import event_monitor as evm
from wazuh_testing.tools import configuration as config
```

Reply:
Done in 5954ee03e8bc85bf442caa23177163d0261e4f09